style.css

```diff
@@ -3,3 +3,842 @@
   padding: 0;
   margin: 0;
 }
+
+html {
+  scroll-behavior: smooth;
+}
+
+@media screen and (max-width: 768px) {
+  nav {
+    padding: 1em;
+    background-color: #f7f7f7;
+  }
+
+  #page__indicator {
+    color: #ec5242;
+  }
+
+  .hero {
+    padding: 6em 1em;
+    background-image: url('assets/images/hands-up.webp');
+    background-repeat: no-repeat;
```

Comment:
Thanks again for the review, but about the background image, I thought it was indicated we could use other images. Plus I can't access the background image because the design is in a picture format.

Reply:
Hi @LIBERCOSOFT,

Note that the change is marked optional and wouldn't hinder the approval of your PR. However, even though the guidelines mentioned an allowance for small changes, you'll agree that the background of the page is a crucial determining factor in the look of the overall website.

Reply:
Alright, thanks.